packages/components/src/components/single-select/shadow.tsx

```diff
@@ -232,7 +232,7 @@ export class KolSingleSelect implements SingleSelectAPI {
 					<div class="kol-single-select__group">
 						<KolInputStateWrapperFc {...this.getInputProps()} />
 
-						{this._inputValue && (
+						{this._inputValue && !this._hideClearButton && (
```

Comment:
Es sollte über den State gegangen werden.
```suggestion
						{this._inputValue && !this.state._hideClearButton && (
```